index.html

```diff
@@ -87,6 +91,8 @@
             <br>
         </div>
         <br>
+        <div class="plyr"><audio></audio></div>
```

Comment:
Chrome. Doesn't work both locally and on local server.  
Also, `<div>` doesn't work either (js throws error). I think when seeing `<div>` Plyr looks for video id and such and can't find it. With `<audio>`, and `<video>` it doesn't care if there is `<source>` inside. Seems like a bug in Plyr for me.